conf/server.properties

```diff
@@ -77,25 +77,25 @@ org.dbpedia.spotlight.spot.ahocorasick.surfaceforms=/data/spotlight/surfaceforms
 # Choose between jdbc or lucene for DBpedia Resource creation. Also, if the jdbc throws an error, lucene will be used.
 org.dbpedia.spotlight.core.database = lucene
 org.dbpedia.spotlight.core.database.jdbcdriver = org.hsqldb.jdbcDriver
-org.dbpedia.spotlight.core.database.connector = jdbc:hsqldb:file:/data/spotlight/3.7/database/spotlight-db;shutdown=true&readonly=true
+org.dbpedia.spotlight.core.database.connector = jdbc:hsqldb:file:data_quickstart/database/spotlight-db;shutdown=true&readonly=true
 org.dbpedia.spotlight.core.database.user = sa
 org.dbpedia.spotlight.core.database.password =
 
 # From http://spotlight.dbpedia.org/download/release-0.5/candidate-index-full.tgz
-org.dbpedia.spotlight.candidateMap.dir = /home/dirk/dbpedia/biomedicine/index-withSF-withTypes
-org.dbpedia.spotlight.candidateMap.loadToMemory = false
+org.dbpedia.spotlight.candidateMap.dir = data_quickstart/index
+org.dbpedia.spotlight.candidateMap.loadToMemory = true
 # Path to Lucene index containing only the candidate map. It is used by document-oriented disambiguators such as Document,TwoStepDisambiguator
 # Only used if one such disambiguator is loaded. Data is at: http://spotlight.dbpedia.org/download/release-0.5/candidate-index-full.tgz
-#org.dbpedia.spotlight.candidateMap.dir = dist/src/deb/control/data/usr/share/dbpedia-spotlight/index
+#org.dbpedia.spotlight.candidateMap.dir = dist/src/deb/control/data_quickstart/usr/share/dbpedia-spotlight/index
 
 
 #----- DISAMBIGUATION -------
 
 # List of disambiguators to load: Document,Occurrences,CuttingEdge,Default
-org.dbpedia.spotlight.disambiguate.disambiguators = CuttingEdge
+org.dbpedia.spotlight.disambiguate.disambiguators = Default,Document
```

Comment:
Honestly, I think this just happened during rebasing somehow. I didn't mean to change anything in that file, I think I'll just reverse this